examples/createProject.ts

```diff
@@ -0,0 +1,33 @@
+import { ImmutableX, Config, CreateProjectRequest } from '@imtbl/core-sdk';
+import { generateWalletConnection } from './libs/walletConnection';
+
+(async () => {
+  const walletConnection = await generateWalletConnection('goerli');
+
+  const imxClient = new ImmutableX(Config.SANDBOX);
+
+  const projectParams: CreateProjectRequest = {
+    company_name: '',
+    contact_email: '',
+    name: '',
+  };
+
+  try {
+    const createProjectResponse = await imxClient.createProject(
+      walletConnection.ethSigner,
+      projectParams,
+    );
+
+    console.log('createProject', JSON.stringify(createProjectResponse));
```

Comment:
Again with the log, can we please update all others?
```suggestion
    console.log('createProjectResponse', JSON.stringify(createProjectResponse));
```

Reply:
[a7f99be](https://github.com/immutable/imx-core-sdk/pull/195/commits/a7f99bee29bf808011d4355d4e30590187b35fcc)